integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonSessionState.scala

```diff
@@ -146,22 +149,31 @@ class CarbonSessionState(sparkSession: SparkSession) extends HiveSessionState(sp
 
   override lazy val optimizer: Optimizer = new CarbonOptimizer(catalog, conf, experimentalMethods)
 
+  def extendedAnalyzerRules: Seq[Rule[LogicalPlan]] = Nil
+  def internalAnalyzerRules: Seq[Rule[LogicalPlan]] = {
```

Comment:
can we rename the method from internalAnalyzerRules to analyzerRules?...As we already have 1 method with name extendedAnalyzerRules, so this name looks more suitable to me

Reply:
ok